Expensify/App
# [No QA] Dont require accessibility checkboxes to ship a deploy checklist

### Details
Coming from https://expensify.slack.com/archives/C07J32337/p1642722197174300

### Fixed Issues
$ n/a

### Tests
1. Close a deploy checklist with the :shipit: comment and all the QA boxes checked but a deploy blocker unchecked.
1. The issue should reopen.
1. Close a deploy checklist with the :shipit: comment and all the deploy blockers checked but a QA box unchecked.
1. The issue should reopen.
1. Close a deploy checklist with the :shipit: comment, all the QA boxes checked, all the deploy blockers checked, and at least one accessibility checkbox unchecked.
1. The issue should not reopen and the production deploy should happen.

### QA Steps
None.

### Tested On
Jest/GitHub only
